.byebug_history

```diff
@@ -0,0 +1,10 @@
+continue
+user.first_name
+n
+user.first_name
+n
+first_name_before
+continue
+User.id
+User
+n
```

Comment:
- byebug_history added to gitignore
- added show action to users
- added tests for user show action